README.md

```diff
@@ -71,6 +71,7 @@ To add your own project below, just [edit](https://github.com/wwdc/2021/edit/mai
 |[Niklas Bülow](https://twitter.com/NiklasBuelow)|[GitHub](https://github.com/insightmind/WWDC21SwiftStudentChallenge)||SpriteKit, UIKit, AVFoundation|Accepted|
 |[Omar Abusharar](http://github.com/omartheturtle)||[Youtube](https://youtu.be/fo5AtVe_PJk)|SwiftUI|Submitted|
 |[Oscar Gorog](https://github.com/OscarGorog)|[GitHub](https://github.com/OscarGorog/WWDC21-Playground)||SwiftUI, Combine, RealityKit, AVFoundation|Rejected|
+|Ozan Mirza|[GitHub](https://github.com/ozanm/WWDC-2021-Neural-Networks)|UIKit, CoreGraphics, Accelerate|Accepted|
```

Comment:
Congrats! 👏
Can you just fix the table real quick? Also, as you probably know, you can link to your name to your Twitter etc. or add a video showing your playground if you want.. ^^
```suggestion
|Ozan Mirza|[GitHub](https://github.com/ozanm/WWDC-2021-Neural-Networks)||UIKit, CoreGraphics, Accelerate|Accepted|
```